static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx

```diff
@@ -50,21 +54,21 @@ export function LineChartListWidget(props: PerformanceWidgetProps) {
   const [selectedListIndex, setSelectListIndex] = useState<number>(0);
   const {ContainerActions} = props;
 
-  if (props.fields.length !== 1) {
-    throw new Error(
-      `Line chart list widget can only accept a single field (${props.fields})`
-    );
-  }
```

Comment:
Should this be put back?

Reply:
Yeah we can now that there is only 1 field per definition still.